src/pages/Login/Login.tsx

```diff
@@ -71,6 +75,25 @@ const RegisterCaption = styled(Caption)`
   margin-bottom: 24px;
 `;
 
+const ErrorField = styled(Field)`
+  display: none;
+`;
+
+const StyledErrorMessage = styled(ErrorMessage)`
+  color: ${props => props.theme.colors.functional.error};
+`;
+
+interface FormValues {
+  email: string;
+  password: string;
+  firebaseError: string;
```

Comment:
why firebaseError is a part of form values? Do You want to submit this error to the backend? Please, don't :)